helios-tools/src/main/java/com/spotify/helios/cli/command/RollingUpdateCommand.java

```diff
@@ -114,28 +112,27 @@ public Long get() {
               + "this will NOT abort the rolling update, it will just cause this command to exit.");
 
     migrateArg = parser.addArgument("--migrate")
-        .setDefault(false)
         .action(storeTrue())
+        .setDefault((Object) null)
```

Comment:
isn't this the default already?

Reply:
`storeTrue()` implicitly sets the default to false

Reply:
It's ugly. Let me know if anyone has a better way to not default to false.

Reply:
ah that is pretty weird. I guess the only other option really is to write your own ArgumentAction that doesn't do that, but that feels silly.